client/extensions/woocommerce/app/settings/taxes/index.js

```diff
@@ -108,7 +108,7 @@ class SettingsTaxes extends Component {
 
 		const onSuccess = () => {
 			this.setState( { isSaving: false, userBeganEditing: false } );
-			return successNotice( translate( 'Settings updated successfully.' ) );
+			return successNotice( translate( 'Settings updated successfully.' ), { duration: 4000 } );
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Theme settings updated successfully.' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/woocommerce-taxes-save-button)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).